.travis.yml

```diff
@@ -2,12 +2,15 @@ language: go
 go:
   - 1.11.x
 sudo: false
+services:
+  - redis-server
 matrix:
   include:
   # Using vendored dependencies
   - install:
     - go get -u golang.org/x/lint/golint
     - go get -u golang.org/x/tools/cmd/...
+    - go get gopkg.in/mikefarah/yq.v2
```

Comment:
shameless selfplug: can we use faq?

```sh
LATEST_FAQ_RELEASE=$(curl -s https://api.github.com/repos/jzelinskie/faq/releases | cat | head -n 10 | grep "tag_name" | cut -d\" -f4)
curl -Lo $GOPATH/bin/faq https://github.com/jzelinskie/faq/releases/download/$LATEST_RELEASE/faq-linux-amd64 && chmod +x $GOPATHbin/faq
```

Reply:
Rather, I'd honestly be more fine with checking in benchmark/testing config files in a separate directory and moving our example config into the documentation directory. It'll avoid having brittle scripts in CI doing find/replace.